stubs/tensorflow/tensorflow/keras/models.pyi

```diff
@@ -0,0 +1,6 @@
+from _typeshed import Incomplete
+
+import tensorflow as tf
+from tensorflow.keras.layers import Layer, _InputT, _OutputT
+
+class Model(Layer[_InputT, _OutputT], tf.Module, Incomplete): ...
```

Comment:
I'd discourage subclassing from Incompete. If your goal is to have Model class defined for callbacks but to not fill it's stub yet (for a future pr), I'd recommend

`Model = Incomplete`

Incomplete is just an alias for Any used for readability. Use Any when there's no good way to type hint it, incomplete when it is not yet done.

Reply:
Future note Model I'd recommend doing `tensorflow.keras.Model.__new__`, `tensorflow.keras.Model.build`, `tensorflow.keras.Model.call` in stubtest allowlist as exceptions for same reason those 3 methods are exceptions for layers [here](https://github.com/python/typeshed/blob/6d9f28a7cae8cdb2721ed18a77e384b8fee4eefd/stubs/tensorflow/%40tests/stubtest_allowlist.txt#L45). Like any other stubtest allowlist exception brief comment is helpful.

Reply:
Understood, thanks.